cnf-certification-test/lifecycle/suite.go

```diff
@@ -484,3 +489,29 @@ func testPodPersistentVolumeReclaimPolicy(env *provider.TestEnvironment) {
 		ginkgo.Fail(errMsg)
 	}
 }
+
+func testCPUIsolation(env *provider.TestEnvironment) {
+	ginkgo.By("Testing pods for CPU isolation requirements")
+
+	// Individual requirements we are looking for:
+	// - Annotations must be provided disabling CPU and IRQ load-balancing.
+	// - Resource Requests and Limits must be provided and identical
+
+	// Additional checks if the above pass
+	// - CPU Requests and Limits must be in the form of whole units
+	// - 'runtimeClassName' must be specified
+
+	podsMissingIsolationRequirements := make(map[string]bool)
+
+	for _, put := range env.Pods {
+		if put.IsCPUIsolationEnabled() && !put.IsCPUIsolationCompliant() {
+			podsMissingIsolationRequirements[put.Data.Name] = true
+		}
+	}
+
+	if n := len(podsMissingIsolationRequirements); n > 0 {
+		errMsg := fmt.Sprintf("Number of pods found that are not compliant with CPU isolation requirements: %d. See logs for more detail.", n)
```

Comment:
Number of pods found -> Number of guaranteed pods found

Reply:
Fixed.